.github/workflows/cypress-test-oidc-e2e.yml

```diff
@@ -223,27 +223,15 @@ jobs:
           EOT
           echo 'HERE IS THE DASHBOARD CONFIG FILE: '
           cat ./config/opensearch_dashboards.yml
-          nohup yarn start --no-base-path --no-watch | tee dashboard.log &
+          nohup ./bin/opensearch-dashboards | tee dashboard.log &
 
       # Check if OSD is ready with a max timeout of 600 seconds
       - name : Check If OpenSearch Dashboards Is Ready
         if: ${{ runner.os == 'Linux' }}
         run: |
-          cd ./OpenSearch-Dashboards
-          echo "Start checking OpenSearch Dashboards."
-          for i in {1..60}; do
-            if grep -q "bundles compiled successfully after" "dashboard.log"; then
-              echo "OpenSearch Dashboards compiled successfully."
-              break
-            fi
-            if [ $i -eq 60 ]; then
-              echo "Timeout for 600 seconds reached. OpenSearch Dashboards did not finish compiling."
-              exit 1
-            fi
-            sleep 10
-          done
+          timeout 300 bash -c 'while [[ "$(curl -u admin:${{ env.OPENSEARCH_INITIAL_ADMIN_PASSWORD }} -k http://localhost:5601/api/status | jq -r '.status.overall.state')" != "green" ]]; do sleep 5; done'
 
       - name: Run Cypress
         run : |
           yarn add cypress --save-dev
-          yarn cypress:run --browser chrome --headless --spec 'test/cypress/e2e/oidc/*.js'
+          yarn cypress:run --browser chromium --headless --spec 'test/cypress/e2e/oidc/*.js'
```

Comment:
nit: just for my knowledge, any specific reason for switching to chromium?

Reply:
I was just testing to see how different browsers are performing, I saw a little flakiness, but I reverted the change now.